osu.Game/Screens/Play/Player.cs

```diff
@@ -630,7 +624,20 @@ public void Restart()
             PerformExit(false);
         }
 
-        private ScheduledDelegate completionProgressDelegate;
+        /// <summary>
+        /// This delegate, when set, means the results screen has been queued to appear.
+        /// The display of the results screen may be delayed by any work being done in <see cref="PrepareScoreForResults"/> and <see cref="PrepareScoreForResultsAsync"/>.
+        /// </summary>
+        /// <remarks>
+        /// Once set, this can *only* be cancelled by rewinding, ie. if ScoreProcessor.HasCompleted becomes <c>false</c>.
```

Comment:
Can see why you omitted the `<see cref=""/>` tag here. You can actually do
```suggestion
        /// Once set, this can *only* be cancelled by rewinding, ie. if <see cref="JudgementProcessor.HasCompleted"/>ScoreProcessor.HasCompleted</see> becomes <see langword="false"/>.
```